addons/website/static/src/client_actions/configurator/configurator.scss

```diff
@@ -27,6 +27,9 @@
             }
         }
 
+        input::placeholder {
+            font-size: 40%;
+        }
```

Comment:
Shouldn't it be moved inside the already existing rule for this input?
https://github.com/odoo/odoo/blob/092a46e3e9dd3a3ec30659b6cebf3a3e63958fb5/addons/website/static/src/client_actions/configurator/configurator.scss#L117

The `40%` is very specific to this input, I doubt it would be reusable for a new input at some point in the configurator pages

Reply:
@rdeodoo
Done sir , please review and decide whether we need to put this in which stable version (15.0, 16.0, or 17.0). Thank you